<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindow</class>
 <widget class="QMainWindow" name="MyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LoginPushButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>330</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>666666</string>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordMindLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsernameLineEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>user3</string>
    </property>
   </widget>
   <widget class="QLabel" name="UsernameMindLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;用户名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ConnectPushButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>再次连接数据库</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelBackGround">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>971</width>
      <height>511</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="DatabaseInfoTextEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;LocalDatabase.db&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>LabelBackGround</zorder>
   <zorder>LoginPushButton</zorder>
   <zorder>PasswordLineEdit</zorder>
   <zorder>PasswordMindLabel</zorder>
   <zorder>UsernameLineEdit</zorder>
   <zorder>UsernameMindLabel</zorder>
   <zorder>ConnectPushButton</zorder>
   <zorder>DatabaseInfoTextEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
